specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-08-01/examples/WafListPolicies.json

```diff
@@ -120,6 +120,16 @@
                 {
                   "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontdoors/fd1/frontendendpoints/fd1-azurefd-net"
                 }
+              ],
+              "routingRuleLinks": [
+                {
+                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontdoors/fd1/routingrules/route"
+                }
+              ],
+              "assignmentPathLinks": [
+                {
+                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontdoors/fd1/assignmentpaths/*/*"
```

Comment:
I don't think we should expose 'assignmentpaths' as a sub-resource of frontdoors. I would just use the FD resource id itself. The user can get the FD resource definition (or any linked resource definition) if the WAF patterns are needed.

Reply:
Removed